src/components/LayerUI.tsx

```diff
@@ -137,7 +137,7 @@ export const LayerUI = React.memo(
               </Section>
               {showSelectedShapeActions(appState, elements) && (
                 <Section
-                  className="App-right-menu"
+                  className="App-menu__left--scrollable"
```

Comment:
```suggestion
                  className="App-menu__left"
```

Reply:
`App-menu__left` class already [exist](https://github.com/excalidraw/excalidraw/blob/master/src/styles.scss#L259-L262). Should I replace it?